src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -35,6 +33,11 @@ export const emailValidation = yup
 
 export const nameValidation = yup.string().required()
 
+export const phoneValidation = yup
+  .string()
+  .required()
+  .min(10)
```

Comment:
Would this work for other markets?

Reply:
we currently don't have the phoneNumber in other markets and only want to put it as part of the checkout in NO. When we release the new checkout this will get removed entirely. 😄